content/en/cosign/verify.md

```diff
@@ -20,7 +40,7 @@ By default, `cosign` validates that this digest matches the container during `co
 If you are using other payload formats with `cosign`, you can use the `--check-claims=false` flag:
 
 ```shell
-$ cosign verify --check-claims=false --key cosign.pub user/demo
+$ cosign verify --check-claims=false user/demo
 Warning: the following claims have not been verified:
 {"Critical":{"Identity":{"docker-reference":""},"Image":{"Docker-manifest-digest":"87ef60f558bad79beea6425a3b28989f01dd417164150ab3baab98dcbf04def8"},"Type":"cosign container image signature"},"Optional":null}
 ```
```

Comment:
Can we add a TODO as a follow up to this PR to add examples for:

* Verifying with a provided key, --key
* Verifying a blob with verify-blob 

No need to add them in this PR to keep it from getting too large, but we should follow up with these missing examples

Reply:
verify-blob example is included now